lib/place_v7.ml

```diff
@@ -0,0 +1,659 @@
+(* Copyright (c) 1998-2007 INRIA *)
+
```

Comment:
we already have place.ml why do we need a place_v7.ml

Reply:
I think Henri replied it was a security not to break place.ml @gnt at all! :)